platform/MCU/rtl872x/src/flash_mal.c

```diff
@@ -553,6 +553,24 @@ static int invalidateUpdateSlot(platform_flash_modules_t* module, size_t offset)
     return dct_write_app_data(&module->magicNumber, offset + offsetof(platform_flash_modules_t, magicNumber), sizeof(module->magicNumber));
 }
 
+int FLASH_AddMfgSystemModuleSlot(void) {
+    int ret = SYSTEM_ERROR_INTERNAL;
+    uint32_t systemPartOffset = MFG_COMBINED_FW_START_ADDRESS + FLASH_ModuleLength(FLASH_INTERNAL, MFG_COMBINED_FW_START_ADDRESS) + 4/*CRC32*/;
```

Comment:
Ideally we should parse the binary to figure out the actual offset of system part1, instead of a fixed relative offset.